apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -207,6 +215,12 @@ class SectionRow extends Component {
     } = this.props;
     const { editing, deleting } = this.state;
 
+    // When deleting a section, I've occasionally seen us attempt a render with
+    // a null section for some reason. This is here to provide defense against this.
+    if (!section) {
+      return null;
+    }
```

Comment:
This was a little bit strange, but then I stopped reproing, so I'm not 100% sure we'll actually ever hit this.

What I think was happening is that we called removeSection, and the connect for`SectionRow` got called before the connect for `SectionTable`. This meant we tried to render a SectionRow with no section, and then immediately did our SectionTable render, in which this SectionRow no longer exists (because we don't have a sectionId for it anymore).

There are other options I could use to deal with this. For example, I could refrain from actually deleting sections from the store (as long as I delete the id from the `sectionIds` list, things should be fine. However, I think the current approach should be fine (though it means we'll hit a propType warning in dev envs if we run into this).